src/server/worker.js

```diff
@@ -44,7 +43,9 @@ export function run(worker) {
 
   // HTTP GraphQL endpoint
   app.post('/graphql', jwt({
-    secret: new Buffer(auth0Secret, 'base64'),
```

Comment:
same as above, let's create the secret, then we can just write `secret` in the object. keeps stuff clean.